scripts/images/convert_qcow2_ova.py

```diff
@@ -128,63 +353,72 @@ def convert_qcow2_ova(imageUrl, imageSize, imageName):
     ova_image_file = tmpdir + '/' + imageName + '.ova'
     ova_gz_image_file = tmpdir + '/' + imageName + '.ova.gz' 
 
-    os.mkdir(converted_images_dir) # Target directory to keep volume, meta and ovf files
+    try:
+        os.mkdir(converted_images_dir) # Target directory to keep volume, meta and ovf files
+        if imageUrl.startswith('http://'):
+            print("Download image.......")
+            get_image(imageUrl, image_file_path )
+        else:
+            shutil.copyfile(imageUrl, image_file_path)
 
-    print("Download image.......")
-    get_image(imageUrl, image_file_path )
+        print("Extracting gz image...")
+        gzip_gunzip(image_file_path, extracted_qcow2_file_path, 65536)
 
-    print("Extracting gz image...")
-    gzip_gunzip(image_file_path, extracted_qcow2_file_path, 65536)
+        print("Converting to raw ....")
+        cmd =  'qemu-img convert -f qcow2 -O raw ' + extracted_qcow2_file_path  + '  ' + extracted_raw_file_path
+        out, err, ret = exec_cmd(cmd)
+        if ret != 0:
+            print('ERROR: problem converting file (do you have qemu-img installed?)')
```

Comment:
probably we can have exit here if it fails, don't proceed further

Reply:
right, will address this..